native/packages/react-native-bpk-component-text-input/src/BpkTextInput.js

```diff
@@ -66,7 +66,7 @@ class BpkTextInput extends Component<Props, State> {
     onFocus: PropTypes.func,
     placeholder: PropTypes.string,
     style: ViewPropTypes.style,
-    valid: PropTypes.oneOf(true, false, null),
+    valid: PropTypes.oneOf([true, false, null]),
```

Comment:
`oneOf` expect an array, this fixed all the warning for TextInput and PhoneInput